whotracksme/data/assets/trackerdb.sql

```diff
@@ -3287,6 +3430,81 @@ INSERT INTO "trackers" VALUES('userzoom.com','UserZoom',3,'https://www.userzoom.
 INSERT INTO "trackers" VALUES('flocktory.com','Flocktory',4,'https://www.flocktory.com/',NULL,'2761',NULL);
 INSERT INTO "trackers" VALUES('messenger.com','Facebook Messenger',7,'https://messenger.com','facebook',NULL,NULL);
 INSERT INTO "trackers" VALUES('ooyala.com','Ooyala',0,'https://www.ooyala.com/','telstra',NULL,NULL);
+INSERT INTO "trackers" VALUES('naver.com','Naver',8,'https://www.naver.com/','naver','2456','humera');
+INSERT INTO "trackers" VALUES('mailerlite.com','Mailerlite',10,'https://www.mailerlite.com/','mailerlite',NULL,'humera - email provider');
+INSERT INTO "trackers" VALUES('mediator.media','Mediator',6,'https://mediator.media/','mycom_bv',NULL,'humera ');
+INSERT INTO "trackers" VALUES('metaffiliation.com','Netaffiliation',4,'http://de.netaffiliation.com/','kwanko',NULL,'humera - metaffiliation redirects to netaffiliation');
+INSERT INTO "trackers" VALUES('mopinion.com','Mopinion',2,'https://mopinion.com/','mopinion',NULL,'humera');
+INSERT INTO "trackers" VALUES('awentw.com','Awentw',3,'https://www.doclerholding.com/en/about/companies/38/','docler ip',NULL,'');
```

Comment:
No, it's a malware. No product or service with the name was found. 
At the moment we've removed this entry because of some other missing information. Will reach out to you for details.